akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/BehaviorTestKitSpec.scala

```diff
@@ -69,12 +69,12 @@ object BehaviorTestKitSpec {
           context.stop(child)
           Behaviors.same
         case SpawnAdapter =>
-          context.spawnMessageAdapter { r: Reproduce =>
+          context.spawnMessageAdapter { (r: Reproduce) =>
```

Comment:
I'd probably use `context.spawnMessageAdapter[Reproduce] { r =>`, but that's for another PR

Reply:
without parentheses will cause error:
```scala
error] -- Error: /Users/thanhbv/ohze/oss/akka/akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/BehaviorTestKitSpec.scala:72:51 
[error] 72 |          context.spawnMessageAdapter[Reproduce] { r: Reproduce =>
[error]    |                                                   ^^^^^^^^^^^^
[error]    |parentheses are required around the parameter of a lambda
[error]    |This construct can be rewritten automatically under -language:Scala2Compat -rewrite.
```